src/test/groovy/io/smartcat/ranger/BuilderMethodsSpec.groovy

```diff
@@ -186,6 +186,7 @@ class BuilderMethodsSpec extends Specification {
 
         where:
         method             | nowString
+        "now"              | new SimpleDateFormat("yyyy-MM-dd").format(new Date())
```

Comment:
Call is:
def gen = new ObjectGeneratorBuilder()
        .prop("today", time("yyyy-MM-dd", BuilderMethods."$method"())).build()
So, now() method will be called inside time method, resulting in string with 'yyyy-MM-dd' format of current time.